client/src/curriculum/utils.ts

```diff
@@ -0,0 +1,74 @@
+import { useEffect } from "react";
+import { CurriculumDoc, CurriculumData } from "../../../libs/types/curriculum";
+import useSWR from "swr";
+import { HTTPError } from "../document";
+import { WRITER_MODE } from "../env";
+
+// Using this import fails the build...
+//import { Topic } from "../../../libs/types/curriculum";
```

Comment:
Is this comment telling us that this enum cannot be defined in `libs/types`, and that's why it's defined here?